app/Console/Commands/TweetImportantCFPDates.php

```diff
@@ -94,12 +95,29 @@ private function tweet($message)
     {
         try {
             $this->twitter->postTweet([
-                'status' => $message
+                'status' => $message,
             ]);
 
-            Log::info('Successfully tweeted: ' . $message);
+            Log::info('Successfully tweeted: '.$message);
         } catch (Exception $e) {
             Log::error($e->getMessage());
         }
     }
+
+    /**
+     * Retrieve only those cfps which should be tweeted.
+     *
+     * @param \Illuminate\Database\Eloquent\Collection $cfps
+     *
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    private function tweetable($cfps)
```

Comment:
Maybe call this `$conferences` to be clearer? And pass `$conference` into the reject callback, just so it's easier to reason with what we're working with.

Great work on this.